### PR TITLE
Even more rewording for the explainer

### PR DESCRIPTION
**Pull request by leobalter**

### CODE REVIEW

**Comment by leobalter** [comment]
@zhangwilling I'm not sure if I understand correctly. Are you saying these updates are being useful? Otherwise, I'd love to improve the docs in a way that works out for all of us.

File: explainer.md [hunk 1]
```diff
@@ -156,9 +156,9 @@ These are some of the key use cases where The Realms API becomes very useful and
 - Template libraries
 - DOM Virtualization
 
-### <a name='ThirdPartyScripts'></a>Third Party Scripts
+### <a name='ThirdPartyScripts'></a>_Trusted_ Third Party Scripts
 
-We acknowledge that applications need a quick and simple execution of Third Party Scripts. There are cases where **many** scripts are executed for the same application. There isn't a need for a new host or agent.
+We acknowledge that applications need a quick and simple execution of Third Party Scripts. There are cases where **many** scripts are executed for the same application. There isn't a need for a new host or agent. This is also not aiming for prevention over non-Trusted Third Party Scripts like malicious code or xss injections. Our focus is on multi libraries and building script blocks from different authors.
```
Comment (caridy) [inline explainer.md:161]: remove the word "script"

File: explainer.md [hunk 2]
```diff
@@ -349,8 +348,29 @@ Using VM module in nodejs, and same-domain iframes in browsers. Although, VM mod
...
+ // attaching the iframe to the DOM tree
+document.body.appendChild(iframe);
+
+var win = iframe.contentWindow;
+
+// Get accessor that returns the topmost window.
+win.top; // Cannot be properly redefined/virtualized: { get: top(), set: undefined, enumerable: true, configurable: false }
+
+// **detaching** the iframe
+document.body.appendChild(iframe);
```
Comment (caridy) [inline explainer.md:375]: removeChild

File: explainer.md [hunk 3]
```diff
@@ -349,8 +348,29 @@ Using VM module in nodejs, and same-domain iframes in browsers. Although, VM mod
...
+
+For clarifications, the term detachable means an iframe pulled out from the DOM tree:
+
+```js
+var iframe = document.createElement("iframe");
+
+ // attaching the iframe to the DOM tree
+document.body.appendChild(iframe);
+
+var win = iframe.contentWindow;
```
Comment (caridy) [inline explainer.md:369]: iframeWindow

**Pull request merged**